### ISSUE TITLE
Allow to define n1_optimized queries with questionmark in the name

### ISSUE THREAD

**Issue by Galathius** [djezzzl/n1_loader#37]
Hello! First and foremost, I want to express my appreciation for your outstanding gem. It has significantly streamlined our workflow.

We recently encountered an unpleasant issue. It appears that we're unable to declare an n1_optimized query with a name that contains a question mark symbol. Here's the use-case:

```ruby
n1_loader :finished? do |reviews|
    finished_review_ids = PerformanceReview::Review.finished.ids
    reviews.each { |review| fulfill(review, finished_review_ids.include?(review.id)) }
end
```

Gem throws the next error:
```
-- `@finished?_loader' is not allowed as an instance variable name
```

This may be a worthwhile improvement to consider.

**Comment by djezzzl** [comment]
Hi @Galathius,

Thank you for your kind words!

I would be happy to improve this. I will do my best to do it this weekend unless you would like to contribute.

In any case, have a great weekend!

**Comment by djezzzl** [comment]
Hi @Galathius,

It was done by: https://github.com/djezzzl/n1_loader/pull/38

It was released in 1.6.4. 

Please try it out and let me know if it works for you. 

Thank you, and have a great day!

**Comment by Galathius** [comment]
Works perfect! Thank you very much! :-)
